assets/src/design-system/components/dropDown/test/utils.js

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { getOptions } from '../utils';
+
+const basicDropDownOptions = [
+  {
+    label: 'label item one',
+    value: 'label-item-one',
+  },
+  {
+    label: 'label item two',
+    value: 'label-item-two',
+  },
+  {
+    label: 'label item three',
+    value: 'label-item-three',
+  },
+];
+
+describe('DropDown/utils getOptions', () => {
+  it('should shape and sanititize basic dropDown options', () => {
+    const groupedOptions = getOptions(basicDropDownOptions);
+
+    expect(groupedOptions).toStrictEqual([
+      {
+        group: [
+          {
+            label: 'label item one',
+            value: 'label-item-one',
+          },
+          {
+            label: 'label item two',
+            value: 'label-item-two',
+          },
+          {
+            label: 'label item three',
+            value: 'label-item-three',
+          },
+        ],
+      },
+    ]);
+  });
+
+  it('should shape and sanititize basic dropDown options even when some data is bad', () => {
+    const groupedOptions = getOptions([
+      ...basicDropDownOptions,
+      'just a string',
+      { label: 'bad data sneaking through', somethingNew: [1, 2, 3] },
```

Comment:
This is good. Sometimes unsanitized data can really break dropdowns. Knowing we eliminate that will help reduce bugs.